csrc/jtag_vpi.c

```diff
@@ -40,7 +40,8 @@
 
 #include <vpi_user.h>
 
-#define RSP_SERVER_PORT	5555
+// Calling bind with port 0 will give return a socket bound to an unused port.
+#define RSP_SERVER_PORT	0
```

Comment:
I think it's only used in one place. In the past, whenever I've forked an external library, I've tried to minimize the changes made to it in order to make it easier to pull in new changes to it.